report/axe.json

```diff
@@ -0,0 +1,886 @@
+{
+  "testSystem": {
+    "name": "axe",
```

Comment:
Lets put in an example project, and leave axe results out until 1) axe-core has done a few more fixes, 2) others have had a chance to work out their own implementation.